backend/Models/usermodel.js

```diff
@@ -1,5 +1,6 @@
 const mongoose = require("mongoose");
 const bcrypt = require("bcrypt");
+const User = require("./")
```

Comment:
What does this code do??

Reply:
idk i didnt write that code. i just got it from main